### PR TITLE
docs: update README to include all components

### PR DESCRIPTION
**Pull request by jessicamcinchak**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```
Comment (jessicamcinchak) [inline .npmrc:1]: fixes pnpm warning
Screenshot from 2022-08-16 11-12-21

File: README.md [hunk 2]
```diff
@@ -1,16 +1,32 @@
-# Map
+# Place components
```
Comment (jessicamcinchak) [inline README.md:1]: Testing out a different name for this thing! How does it feel? See #22

File: README.md [hunk 3]
```diff
@@ -1,16 +1,32 @@
...
 
 [![npm @opensystemslab/map](https://img.shields.io/npm/v/@opensystemslab/map?style=flat-square)](http://npm.im/@opensystemslab/map)
 
-An [OpenLayers](https://openlayers.org/)-powered [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) map for tasks related to planning permission in the UK.
+A library of [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) for tasks related to addresses and planning permission in the UK built with [Lit](https://lit.dev/), [Vite](https://vitejs.dev/), and [Ordnance Survey APIs](https://developer.ordnancesurvey.co.uk/).
+
+***Web map***
```
Comment (jessicamcinchak) [inline README.md:7]: Again, related to #22, maybe in another PR I'll follow through with a proper rename from `my-map` to `web-map` - any other ideas??

File: README.md (hunk 2 above)
Comment (DafyddLlyr) [reply README.md:1]: Yeah I picked up on this change in the previous Pitsby PR - it works for me 👍  Other options would be `location`, `gis`, `geospatial`, `spatial`.

Let's go with Place!

File: README.md (hunk 3 above)
Comment (DafyddLlyr) [reply README.md:7]: I'd support that - makes a lot more sense than `my-map`.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,32 @@
...
+`<address-autocomplete />` fetches addresses in a given UK postcode using the [OS Places API](https://developer.ordnancesurvey.co.uk/os-places-api) and displays them using GOV.UK's [accessible-autocomplete](https://github.com/alphagov/accessible-autocomplete) component. An event is dispatched with the OS record when you select an address.
+
+These web components can be used independently or together following GOV.UK's [Address lookup](https://design-system.service.gov.uk/patterns/addresses/) design pattern.
+
+## Documentation & examples
+
+- Interactive web component docs [oslmap.netlify.app](https://oslmap.netlify.app)
 - [CodeSandbox](https://codesandbox.io/s/confident-benz-rr0s9?file=/index.html) (note: update the CDN script with a version number for new features)
 
+Find these components in the wild, including what we're learning through public beta user-testing, at [https://www.ripa.digital/](https://www.ripa.digital/).
```
Comment (DafyddLlyr) [inline README.md:28]: This is great to include.

**Review by DafyddLlyr — APPROVED** [review]
Great stuff! 👍

**Pull request merged**